src/main/java/ru/sbt/mipt/oop/Application.java

```diff
@@ -1,80 +1,31 @@
 package ru.sbt.mipt.oop;
 
-import com.google.gson.Gson;
-
 import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-
-import static ru.sbt.mipt.oop.SensorEventType.*;
+import java.util.ArrayList;
+import java.util.List;
 
 public class Application {
 
     public static void main(String... args) throws IOException {
         // считываем состояние дома из файла
-        Gson gson = new Gson();
-        String json = new String(Files.readAllBytes(Paths.get("smart-home-1.js")));
-        SmartHome smartHome = gson.fromJson(json, SmartHome.class);
+        SmartHomeReader reader = new SmartHomeJSONReader();
+        String source = "smart-home-1.js";
+        SmartHome smartHome = reader.readSmartHome(source);
         // начинаем цикл обработки событий
-        SensorEvent event = getNextSensorEvent();
+        SensorEventProvider sensor = new RandomSensorEventProvider();
+        Signalization signalization = new Signalization();
+        signalization.setState(new Deactivated(signalization));
+        List<EventProcessor> processors = new ArrayList<>();
+        processors.add(new DoorEventProcessor(smartHome));
+        processors.add(new LightEventProcessor(smartHome));
+        processors.add(new HallDoorEventProcessor(smartHome));
+        processors.add(new SignalizationEventProcessor(signalization));
+        ProcessingScript processor = new StandardProcessingScript(processors);
+        SensorEvent event = sensor.getNextSensorEvent();
         while (event != null) {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BF%D0%B5%D1%80%D0%B5%D0%BD%D0%B5%D1%81%D0%B8%D1%82%D0%B5-%D0%BB%D0%BE%D0%B3%D0%B8%D0%BA%D1%83-%D1%86%D0%B8%D0%BA%D0%BB%D0%B0-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D0%B8-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%B2-%D0%BE%D1%82%D0%B4%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9-%D0%BA%D0%BB%D0%B0%D1%81%D1%81